lib/modem_antenna/Kconfig

```diff
@@ -25,16 +27,16 @@ endchoice
 
 config MODEM_ANTENNA_AT_MAGPIO
 	string "AT%XMAGPIO command"
-	default "AT\%XMAGPIO=1,0,0,1,1,1574,1577" if BOARD_NRF9160DK_NRF9160_NS
+	default "AT\%XMAGPIO=1,0,0,1,1,1574,1577" if (BOARD_NRF9160DK_NRF9160_NS || BOARD_NRF9161DK_NRF9161_NS)
 	default "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200,3,824,894,4,880,960,5,791,849,7,1565,1586" if BOARD_THINGY91_NRF9160_NS
 	help
 	  Defines the AT%XMAGPIO command to be sent to the modem. Leave
 	  empty if this command should not be sent.
 
 config MODEM_ANTENNA_AT_COEX0
 	string "AT%XCOEX0 command"
-	default "AT\%XCOEX0=1,1,1565,1586" if (BOARD_NRF9160DK_NRF9160_NS || BOARD_THINGY91_NRF9160_NS) && MODEM_ANTENNA_GNSS_ONBOARD
-	default "AT\%XCOEX0" if (BOARD_NRF9160DK_NRF9160_NS || BOARD_THINGY91_NRF9160_NS) && MODEM_ANTENNA_GNSS_EXTERNAL
+	default "AT\%XCOEX0=1,1,1565,1586" if (SOC_SERIES_NRF91X && MODEM_ANTENNA_GNSS_ONBOARD)
+	default "AT\%XCOEX0" if (SOC_SERIES_NRF91X && MODEM_ANTENNA_GNSS_EXTERNAL)
```

Comment:
I think we could just remove `SOC_SERIES_NRF91X` from these conditions. I probably put the DK/Thingy flagging here myself originally, but I don't think it's really needed. When the library is enabled, the default COEX0 configuration can be selected based on the type of the antenna.